cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/scorecompiler/ExecutionStepFactory.java

```diff
@@ -91,17 +91,19 @@ public ExecutionStep createActionStep(Long index, Map<String, Serializable> acti
         Map<String, Serializable> actionData = new HashMap<>();
         ActionType actionType;
 
-        @SuppressWarnings("unchecked") Map<String, String> javaActionData = (Map<String, String>) actionRawData.get(SlangTextualKeys.JAVA_ACTION);
-        @SuppressWarnings("unchecked") String pythonScript = (String) actionRawData.get(ScoreLangConstants.PYTHON_SCRIPT_KEY);
+        @SuppressWarnings("unchecked")
+        Map<String, String> javaActionData = (Map<String, String>) actionRawData.get(SlangTextualKeys.JAVA_ACTION_KEY);
+        @SuppressWarnings("unchecked")
+        Map<String, Serializable> pythonActionData = (Map<String, Serializable>) actionRawData.get(SlangTextualKeys.PYTHON_ACTION_KEY);
```

Comment:
python action data contains the script (string) but also dependencies (which will be list -- multiple py dependencies)